nms/app/components/TopBar.tsx

```diff
@@ -123,6 +116,7 @@ export default function TopBar(props: Props) {
                 textColor="inherit"
                 className={classes.tabs}>
                 {props.tabs.map(tab => (
+                  // @ts-ignore
```

Comment:
Should one be a bit more specific about which rule to ignore here? I believe all other ignores do so.

Reply:
This is not about ignoring a rule, it is not for a linter it is for the TypeScript compiler and `ts-ignore` doesn't have the options to only ignore certain things.